controller/apps_kubeclient.go

```diff
@@ -0,0 +1,717 @@
+package controller
```

Comment:
This would better fit in a `kubernetes` pkg, IMO

Reply:
+1. We should keep only controllers in the controller package.

Reply:
I'll move both auth_kubeclient and auth_metrics.